aws/resource_aws_glue_security_configuration.go

```diff
@@ -272,8 +272,10 @@ func expandGlueS3Encryption(m map[string]interface{}) *glue.S3Encryption {
 		S3EncryptionMode: aws.String(m["s3_encryption_mode"].(string)),
 	}
 
-	if v, ok := m["kms_key_arn"]; ok {
```

Comment:
Hi @maparent 👋 Does it work if we instead do a `v.(string) != ""` check here? Introducing the additional `*s3Encryption.S3EncryptionMode != glue.S3EncryptionModeSseS3` conditional can lead to confusing perpetual differences (and if the configuration of both with real values is invalid, the API error can be returned)

```go
if v, ok := m["kms_key_arn"]; ok && v.(string) != "" {
```

Reply:
Yes sounds like a great solution, sorry I missed it.

Reply:
Not at my computer, will alter the pr in 1h if you can’t.